redpanda/templates/configmap.yaml

```diff
@@ -64,7 +64,7 @@ data:
   {{- end }}
 {{- end }}
     redpanda:
-{{- if not (include "redpanda.semver" . | semverCompare ">=22.1.1") }}
+{{- if (include "redpanda-atleast-22-1-1" . | fromJson).bool }}
```

Comment:
@joejulian This is the line that causes this commit to fail linting with the following error:

```
[ERROR] templates/: template: redpanda/templates/statefulset.yaml:55:28: executing "redpanda/templates/statefulset.yaml" at <include (print $.Template.BasePath "/configmap.yaml") .>: error calling include: template: redpanda/templates/configmap.yaml:78:21: executing "redpanda/templates/configmap.yaml" at <"storage_min_free_bytes">: can't handle "storage_min_free_bytes" for arg of type map[string]interface {}
```
I think the error doesn't accurately point to the issue. If this line is reverted then the linting passes. But this line is identical to line 40 above which passes. Do you have an idea as to why?